src/web/components/elements/BlockquoteBlockComponent.tsx

```diff
@@ -1,52 +1,43 @@
-import { css, ClassNames } from '@emotion/react';
+import { ClassNames } from '@emotion/react';
 
 import { body } from '@guardian/src-foundations/typography';
+
+import { QuoteIcon } from '@root/src/web/components/QuoteIcon';
 import { unwrapHtml } from '@root/src/model/unwrapHtml';
 import { RewrappedComponent } from '@root/src/web/components/elements/RewrappedComponent';
-import { QuoteIcon } from '@root/src/web/components/QuoteIcon';
 
 type Props = {
 	html: string;
 	palette: Palette;
 	quoted?: boolean;
+	format: Format;
 };
 
-const BlockquoteRow = ({ children }: { children: React.ReactNode }) => (
-	<blockquote
-		css={css`
-			display: flex;
-			flex-direction: row;
-		`}
-	>
-		{children}
-	</blockquote>
-);
-
```

Comment:
no need for this abstraction, l plus it allows us to just use `css` from `ClassNames` without aliasing the name